### ISSUE TITLE
Bot menu not showing

### ISSUE THREAD

**Issue by uurha** [telegramdesktop/tdesktop#26032]
### Steps to reproduce

1. Own group where more than 200 users
2. Add bot to chat owned by you
3. Try type / or press [/] button on right side near send button.

### Expected behaviour

Bot commands appear as list.
![image](https://user-images.githubusercontent.com/22265817/224864942-91a00cfa-321a-403a-906b-145adec279e5.png)

### Actual behaviour

Nothing happens
![image](https://user-images.githubusercontent.com/22265817/224865344-d05de967-8d39-41b2-9a70-0608db346bb0.png)

### Operating system

Win 10

### Version of Telegram Desktop

4.6.5 x64

### Installation source

Static binary from official website

### Logs

```shell
Here logs where I tried first open bot menu in chat where 2 users (where bot command list is working) and then open bot commands list in chat where more than 200 users

[02:06:01.260 00-0001002] MTP Info: unregistering request 2153. (mtp_instance.cpp : 25)
[02:06:01.260 00-0001003] MTP Info: adding request to toSendMap, msCanWait 0 (session.cpp : 25)
[02:06:01.260 00-0001004] MTP Info: added, requestId 2156 (session.cpp : 25)
[02:06:01.262 00-0001005] MTP Info: dcWithShift 2 stopped send timer, can wait for 0ms from current 894794 (session.cpp : 25)
[02:06:01.262 04-0001006] MTP Info: tryToSend for dc 2. (session_private.cpp : 25)
[02:06:01.262 04-0001007] MTP Info: dc 2 trying to send after ping, state: 2 (session_private.cpp : 25)
[02:06:01.262 04-0001010] MTP Info: sending request, size: 50, num: 531212832, time: 1678755963 (session_private.cpp : 25)
[02:06:01.262 04-0001011] Connection 238(dc:2,149.154.167.41:443) Info: TCP Info: write packet 201 bytes. (connection_abstract.cpp : 25)
[02:06:01.293 04-0001012] Connection 238(dc:2,149.154.167.41:443) Info: Read 89 bytes (connection_abstract.cpp : 25)
[02:06:01.293 04-0001013] Connection 238(dc:2,149.154.167.41:443) Info: Packet received, size = 88. (connection_abstract.cpp : 25)
[02:06:01.293 04-0001014] Connection 238(dc:2,149.154.167.41:443) Info: Decrypted message 7210201956858595329,685,[TRUE] is 48 len (connection_abstract.cpp : 25)
[02:06:01.293 04-0001016] RPC Info: response received for 7210201959581198880, queueing... (session_private.cpp : 25)
[02:06:01.293 04-0001017] MTP Info: MTProtoDC::connectionWasInited(), dcWithShift 2 (session.cpp : 25)
[02:06:01.293 04-0001018] Message Info: requests acked, ids [7210201959581198880, 7210201959581198880] (session_private.cpp : 25)
[02:06:01.293 04-0001019] Message Info: removing some old acked sent msgIds 1 (session_private.cpp : 25)
[02:06:01.293 04-0001020] MTP Info: will send 1 acks, ids: [7210201956858595329, 7210201956858595329] (session_private.cpp : 25)
[02:06:01.293 04-0001021] MTP Info: queueTryToReceive() - need to parse in another thread, 1 messages. (session_private.cpp : 25)
[02:06:01.293 00-0001022] MTP Info: dcWithShift 2 can wait for 10000ms from current 0 (session.cpp : 25)
[02:06:01.293 00-0001023] RPC Info: found parser for request 2156, trying to parse response... (mtp_instance.cpp : 25)
[02:06:01.293 00-0001024] MTP Info: unregistering request 2156. (mtp_instance.cpp : 25)
[02:06:02.128 00-0001025] MTP Info: Cancel request 2156. (mtp_instance.cpp : 25)
[02:06:02.128 00-0001026] MTP Info: unregistering request 2156. (mtp_instance.cpp : 25)
[02:06:02.128 00-0001027] MTP Info: adding request to toSendMap, msCanWait 0 (session.cpp : 25)
[02:06:02.128 00-0001028] MTP Info: added, requestId 2157 (session.cpp : 25)
[02:06:02.129 00-0001029] QTextCursor::setPosition: Position '-1' out of range
[02:06:02.130 00-0001030] MTP Info: dcWithShift 2 stopped send timer, can wait for 0ms from current 900481 (session.cpp : 25)
[02:06:02.130 04-0001031] MTP Info: tryToSend for dc 2. (session_private.cpp : 25)
[02:06:02.130 04-0001032] MTP Info: dc 2 trying to send after ping, state: 2 (session_private.cpp : 25)
[02:06:02.130 04-0001035] MTP Info: sending request, size: 70, num: -34729620, time: 1678755963 (session_private.cpp : 25)
[02:06:02.130 04-0001036] Connection 238(dc:2,149.154.167.41:443) Info: TCP Info: write packet 281 bytes. (connection_abstract.cpp : 25)
[02:06:02.131 00-0001037] MTP Info: adding request to toSendMap, msCanWait 0 (session.cpp : 25)
[02:06:02.131 00-0001038] MTP Info: added, requestId 2158 (session.cpp : 25)
[02:06:02.132 00-0001039] MTP Info: dcWithShift 2 stopped send timer, can wait for 0ms from current 0 (session.cpp : 25)
[02:06:02.132 04-0001040] MTP Info: tryToSend for dc 2. (session_private.cpp : 25)
[02:06:02.132 04-0001041] MTP Info: dc 2 trying to send after ping, state: 2 (session_private.cpp : 25)
[02:06:02.132 04-0001044] MTP Info: sending request, size: 30, num: -26895284, time: 1678755963 (session_private.cpp : 25)
[02:06:02.132 04-0001045] Connection 238(dc:2,149.154.167.41:443) Info: TCP Info: write packet 121 bytes. (connection_abstract.cpp : 25)
[02:06:02.162 04-0001046] Connection 238(dc:2,149.154.167.41:443) Info: Read 89 bytes (connection_abstract.cpp : 25)
[02:06:02.162 04-0001047] Connection 238(dc:2,149.154.167.41:443) Info: Packet received, size = 88. (connection_abstract.cpp : 25)
[02:06:02.162 04-0001048] Connection 238(dc:2,149.154.167.41:443) Info: Decrypted message 7210201960588523521,687,[TRUE] is 48 len (connection_abstract.cpp : 25)
[02:06:02.162 04-0001050] RPC Info: response received for 7210201963310160580, queueing... (session_private.cpp : 25)
[02:06:02.162 04-0001051] MTP Info: MTProtoDC::connectionWasInited(), dcWithShift 2 (session.cpp : 25)
[02:06:02.162 04-0001052] Message Info: requests acked, ids [7210201963310160580, 7210201963310160580] (session_private.cpp : 25)
[02:06:02.162 04-0001053] Message Info: removing some old acked sent msgIds 1 (session_private.cpp : 25)
[02:06:02.162 04-0001054] MTP Info: will send 1 acks, ids: [7210201960588523521, 7210201960588523521] (session_private.cpp : 25)
[02:06:02.162 04-0001055] MTP Info: queueTryToReceive() - need to parse in another thread, 1 messages. (session_private.cpp : 25)
[02:06:02.162 00-0001056] MTP Info: dcWithShift 2 can wait for 10000ms from current 0 (session.cpp : 25)
[02:06:02.162 00-0001057] RPC Info: found parser for request 2157, trying to parse response... (mtp_instance.cpp : 25)
[02:06:02.162 00-0001058] MTP Info: unregistering request 2157. (mtp_instance.cpp : 25)
[02:06:02.181 04-0001059] Connection 238(dc:2,149.154.167.41:443) Info: Read 89 bytes (connection_abstract.cpp : 25)
[02:06:02.181 04-0001060] Connection 238(dc:2,149.154.167.41:443) Info: Packet received, size = 88. (connection_abstract.cpp : 25)
[02:06:02.181 04-0001061] Connection 238(dc:2,149.154.167.41:443) Info: Decrypted message 7210201960674617345,689,[TRUE] is 48 len (connection_abstract.cpp : 25)
[02:06:02.181 04-0001063] RPC Info: response received for 7210201963318058060, queueing... (session_private.cpp : 25)
[02:06:02.181 04-0001064] MTP Info: MTProtoDC::connectionWasInited(), dcWithShift 2 (session.cpp : 25)
[02:06:02.181 04-0001065] Message Info: requests acked, ids [7210201963318058060, 7210201963318058060] (session_private.cpp : 25)
[02:06:02.181 04-0001066] Message Info: removing some old acked sent msgIds 1 (session_private.cpp : 25)
[02:06:02.181 04-0001067] MTP Info: will send 2 acks, ids: [7210201960588523521, 7210201960588523521, 7210201960674617345] (session_private.cpp : 25)
[02:06:02.181 04-0001068] MTP Info: queueTryToReceive() - need to parse in another thread, 1 messages. (session_private.cpp : 25)
[02:06:02.181 00-0001069] MTP Info: dcWithShift 2 can wait for 9980ms from current 901350 (session.cpp : 25)
[02:06:02.182 00-0001070] RPC Info: found parser for request 2158, trying to parse response... (mtp_instance.cpp : 25)
[02:06:02.182 00-0001071] MTP Info: unregistering request 2158. (mtp_instance.cpp : 25)
[02:06:02.259 00-0001072] MTP Info: adding request to toSendMap, msCanWait 0 (session.cpp : 25)
[02:06:02.259 00-0001073] MTP Info: added, requestId 2159 (session.cpp : 25)
[02:06:02.259 00-0001074] MTP Info: dcWithShift 2 stopped send timer, can wait for 0ms from current 901370 (session.cpp : 25)
[02:06:02.259 04-0001075] MTP Info: tryToSend for dc 2. (session_private.cpp : 25)
[02:06:02.259 04-0001076] MTP Info: dc 2 trying to send after ping, state: 2 (session_private.cpp : 25)
[02:06:02.259 04-0001079] MTP Info: sending request, size: 102, num: 520605920, time: 1678755964 (session_private.cpp : 25)
[02:06:02.259 04-0001080] Connection 238(dc:2,149.154.167.41:443) Info: TCP Info: write packet 409 bytes. (connection_abstract.cpp : 25)
[02:06:02.280 04-0001081] Connection 238(dc:2,149.154.167.41:443) Info: Read 89 bytes (connection_abstract.cpp : 25)
[02:06:02.280 04-0001082] Connection 238(dc:2,149.154.167.41:443) Info: Packet received, size = 88. (connection_abstract.cpp : 25)
[02:06:02.280 04-0001083] Connection 238(dc:2,149.154.167.41:443) Info: Decrypted message 7210201961098774529,691,[TRUE] is 48 len (connection_abstract.cpp : 25)
[02:06:02.280 04-0001085] RPC Info: response received for 7210201963865466916, queueing... (session_private.cpp : 25)
[02:06:02.280 04-0001086] MTP Info: MTProtoDC::connectionWasInited(), dcWithShift 2 (session.cpp : 25)
[02:06:02.280 04-0001087] Message Info: requests acked, ids [7210201963865466916, 7210201963865466916] (session_private.cpp : 25)
[02:06:02.280 04-0001088] Message Info: removing some old acked sent msgIds 1 (session_private.cpp : 25)
[02:06:02.280 04-0001089] MTP Info: will send 1 acks, ids: [7210201961098774529, 7210201961098774529] (session_private.cpp : 25)
[02:06:02.280 04-0001090] MTP Info: queueTryToReceive() - need to parse in another thread, 1 messages. (session_private.cpp : 25)
[02:06:02.280 00-0001091] MTP Info: dcWithShift 2 can wait for 10000ms from current 0 (session.cpp : 25)
[02:06:02.280 00-0001092] RPC Info: found parser for request 2159, trying to parse response... (mtp_instance.cpp : 25)
[02:06:02.280 00-0001093] MTP Info: unregistering request 2159. (mtp_instance.cpp : 25)
[02:06:02.491 00-0001094] MTP Info: adding request to toSendMap, msCanWait 0 (session.cpp : 25)
[02:06:02.491 00-0001095] MTP Info: added, requestId 2160 (session.cpp : 25)
[02:06:02.491 00-0001096] MTP Info: dcWithShift 4 stopped send timer, can wait for 0ms from current 0 (session.cpp : 25)
[02:06:02.491 05-0001097] MTP Info: tryToSend for dc 4. (session_private.cpp : 25)
[02:06:02.491 05-0001098] MTP Info: dc 4 trying to send after ping, state: 2 (session_private.cpp : 25)
[02:06:02.491 05-0001101] MTP Info: sending request, size: 22, num: 1516197652, time: 1678755964 (session_private.cpp : 25)
[02:06:02.491 05-0001102] Connection 5(dc:4,149.154.167.92:443) Info: TCP Info: write packet 89 bytes. (connection_abstract.cpp : 25)
[02:06:02.514 05-0001103] Connection 5(dc:4,149.154.167.92:443) Info: Read 89 bytes (connection_abstract.cpp : 25)
[02:06:02.514 05-0001104] Connection 5(dc:4,149.154.167.92:443) Info: Packet received, size = 88. (connection_abstract.cpp : 25)
[02:06:02.514 05-0001105] Connection 5(dc:4,149.154.167.92:443) Info: Decrypted message 7210201962108816385,377,[TRUE] is 52 len (connection_abstract.cpp : 25)
[02:06:02.514 05-0001107] RPC Info: response received for 7210201964861150996, queueing... (session_private.cpp : 25)
[02:06:02.514 05-0001108] MTP Info: MTProtoDC::connectionWasInited(), dcWithShift 4 (session.cpp : 25)
[02:06:02.514 05-0001109] Message Info: requests acked, ids [7210201964861150996, 7210201964861150996] (session_private.cpp : 25)
[02:06:02.514 05-0001110] MTP Info: will send 1 acks, ids: [7210201962108816385, 7210201962108816385] (session_private.cpp : 25)
[02:06:02.514 05-0001111] MTP Info: queueTryToReceive() - need to parse in another thread, 1 messages. (session_private.cpp : 25)
[02:06:02.514 00-0001112] MTP Info: dcWithShift 4 can wait for 10000ms from current 0 (session.cpp : 25)
[02:06:02.514 00-0001113] RPC Info: found parser for request 2160, trying to parse response... (mtp_instance.cpp : 25)
[02:06:02.514 00-0001114] MTP Info: unregistering request 2160. (mtp_instance.cpp : 25)
[02:06:03.819 00-0001115] MTP Info: adding request to toSendMap, msCanWait 0 (session.cpp : 25)
[02:06:03.819 00-0001116] MTP Info: added, requestId 2161 (session.cpp : 25)
[02:06:03.819 00-0001117] MTP Info: dcWithShift 2 stopped send timer, can wait for 0ms from current 901469 (session.cpp : 25)
[02:06:03.819 04-0001118] MTP Info: tryToSend for dc 2. (session_private.cpp : 25)
[02:06:03.819 04-0001119] MTP Info: dc 2 trying to send after ping, state: 2 (session_private.cpp : 25)
[02:06:03.819 04-0001122] MTP Info: sending request, size: 70, num: -1371296652, time: 1678755965 (session_private.cpp : 25)
[02:06:03.819 04-0001123] Connection 238(dc:2,149.154.167.41:443) Info: TCP Info: write packet 281 bytes. (connection_abstract.cpp : 25)
[02:06:03.858 04-0001124] Connection 238(dc:2,149.154.167.41:443) Info: Read 121 bytes (connection_abstract.cpp : 25)
[02:06:03.858 04-0001125] Connection 238(dc:2,149.154.167.41:443) Info: Packet received, size = 120. (connection_abstract.cpp : 25)
[02:06:03.858 04-0001126] Connection 238(dc:2,149.154.167.41:443) Info: Decrypted message 7210201967790982145,693,[TRUE] is 80 len (connection_abstract.cpp : 25)
[02:06:03.858 04-0001128] RPC Info: response received for 7210201970563514400, queueing... (session_private.cpp : 25)
[02:06:03.858 04-0001129] MTP Info: MTProtoDC::connectionWasInited(), dcWithShift 2 (session.cpp : 25)
[02:06:03.858 04-0001130] Message Info: requests acked, ids [7210201970563514400, 7210201970563514400] (session_private.cpp : 25)
[02:06:03.858 04-0001131] Message Info: removing some old acked sent msgIds 1 (session_private.cpp : 25)
[02:06:03.858 04-0001132] MTP Info: will send 1 acks, ids: [7210201967790982145, 7210201967790982145] (session_private.cpp : 25)
[02:06:03.858 04-0001133] MTP Info: queueTryToReceive() - need to parse in another thread, 1 messages. (session_private.cpp : 25)
[02:06:03.859 00-0001134] MTP Info: dcWithShift 2 can wait for 10000ms from current 0 (session.cpp : 25)
[02:06:03.859 00-0001135] RPC Info: found parser for request 2161, trying to parse response... (mtp_instance.cpp : 25)
[02:06:03.859 00-0001136] MTP Info: unregistering request 2161. (mtp_instance.cpp : 25)
[02:06:04.613 00-0001137] MTP Info: adding request to toSendMap, msCanWait 0 (session.cpp : 25)
[02:06:04.613 00-0001138] MTP Info: added, requestId 2162 (session.cpp : 25)
[02:06:04.614 00-0001139] Group Call Participants: Reloading with queued: 0 (data_group_call.cpp : 25)
[02:06:04.614 00-0001140] MTP Info: adding request to toSendMap, msCanWait 0 (session.cpp : 25)
[02:06:04.614 00-0001141] MTP Info: added, requestId 2163 (session.cpp : 25)
[02:06:04.614 00-0001142] MTP Info: dcWithShift 2 stopped send timer, can wait for 0ms from current 903047 (session.cpp : 25)
[02:06:04.614 00-0001143] MTP Info: dcWithShift 2 stopped send timer, can wait for 0ms from current 0 (session.cpp : 25)
[02:06:04.615 04-0001144] MTP Info: tryToSend for dc 2. (session_private.cpp : 25)
[02:06:04.615 04-0001145] MTP Info: dc 2 trying to send after ping, state: 2 (session_private.cpp : 25)
[02:06:04.615 04-0001149] MTP Info: sending request, size: 94, num: 2047532784, time: 1678755966 (session_private.cpp : 25)
[02:06:04.615 04-0001150] Connection 238(dc:2,149.154.167.41:443) Info: TCP Info: write packet 377 bytes. (connection_abstract.cpp : 25)
[02:06:04.615 04-0001151] MTP Info: tryToSend for dc 2. (session_private.cpp : 25)
[02:06:04.615 04-0001152] MTP Info: dc 2 trying to send after ping, state: 2 (session_private.cpp : 25)
[02:06:04.637 04-0001153] Connection 238(dc:2,149.154.167.41:443) Info: Read 217 bytes (connection_abstract.cpp : 25)
[02:06:04.637 04-0001154] Connection 238(dc:2,149.154.167.41:443) Info: Packet received, size = 216. (connection_abstract.cpp : 25)
[02:06:04.637 04-0001155] Connection 238(dc:2,149.154.167.41:443) Info: Decrypted message 7210201971224210433,696,[FALSE] is 172 len (connection_abstract.cpp : 25)
[02:06:04.637 04-0001157] Message Info: container received, count: 2 (session_private.cpp : 25)
[02:06:04.637 04-0001158] Message Info: message from container, msg_id: 7210201971224087553, needAck: [TRUE] (session_private.cpp : 25)
[02:06:04.637 04-0001159] RPC Info: response received for 7210201973982380340, queueing... (session_private.cpp : 25)
[02:06:04.637 04-0001160] MTP Info: MTProtoDC::connectionWasInited(), dcWithShift 2 (session.cpp : 25)
[02:06:04.637 04-0001161] Message Info: requests acked, ids [7210201973982380340, 7210201973982380340] (session_private.cpp : 25)
[02:06:04.637 04-0001162] Message Info: removing some old acked sent msgIds 1 (session_private.cpp : 25)
[02:06:04.637 04-0001163] Message Info: message from container, msg_id: 7210201971224198145, needAck: [FALSE] (session_private.cpp : 25)
[02:06:04.637 04-0001164] Message Info: acks received, ids: [7210201973982310756, 7210201973982310756] (session_private.cpp : 25)
[02:06:04.637 04-0001165] Message Info: requests acked, ids [7210201973982310756, 7210201973982310756] (session_private.cpp : 25)
[02:06:04.637 04-0001166] Message Info: ignoring ACK for msgId 7210201973982310756 because request 2162 requires a response (session_private.cpp : 25)
[02:06:04.637 04-0001167] MTP Info: will send 1 acks, ids: [7210201971224087553, 7210201971224087553] (session_private.cpp : 25)
[02:06:04.637 04-0001168] MTP Info: queueTryToReceive() - need to parse in another thread, 1 messages. (session_private.cpp : 25)
[02:06:04.637 00-0001169] MTP Info: dcWithShift 2 can wait for 10000ms from current 0 (session.cpp : 25)
[02:06:04.638 00-0001170] RPC Info: found parser for request 2163, trying to parse response... (mtp_instance.cpp : 25)
[02:06:04.638 00-0001171] Group Call Participants: Set from groupCall 5 -> 5 (data_group_call.cpp : 25)
[02:06:04.638 00-0001172] MTP Info: unregistering request 2163. (mtp_instance.cpp : 25)
[02:06:04.662 04-0001173] Connection 238(dc:2,149.154.167.41:443) Info: Read 1852 bytes (connection_abstract.cpp : 25)
[02:06:04.662 04-0001174] Connection 238(dc:2,149.154.167.41:443) Info: Packet received, size = 1848. (connection_abstract.cpp : 25)
[02:06:04.662 04-0001175] Connection 238(dc:2,149.154.167.41:443) Info: Decrypted message 7210201971328113665,697,[TRUE] is 1800 len (connection_abstract.cpp : 25)
[02:06:04.662 04-0001177] RPC Info: response received for 7210201973982310756, queueing... (session_private.cpp : 25)
[02:06:04.662 04-0001178] RPC Info: gzip container (session_private.cpp : 25)
[02:06:04.662 04-0001179] MTP Info: MTProtoDC::connectionWasInited(), dcWithShift 2 (session.cpp : 25)
[02:06:04.662 04-0001180] Message Info: requests acked, ids [7210201973982310756, 7210201973982310756] (session_private.cpp : 25)
[02:06:04.662 04-0001181] Message Info: removing some old acked sent msgIds 1 (session_private.cpp : 25)
[02:06:04.662 04-0001182] MTP Info: will send 2 acks, ids: [7210201971224087553, 7210201971224087553, 7210201971328113665] (session_private.cpp : 25)
[02:06:04.662 04-0001183] MTP Info: queueTryToReceive() - need to parse in another thread, 1 messages. (session_private.cpp : 25)
[02:06:04.663 00-0001184] MTP Info: dcWithShift 2 can wait for 9975ms from current 903825 (session.cpp : 25)
[02:06:04.663 00-0001185] RPC Info: found parser for request 2162, trying to parse response... (mtp_instance.cpp : 25)
[02:06:04.663 00-0001186] MTP Info: unregistering request 2162. (mtp_instance.cpp : 25)
[02:06:05.238 00-0001187] QTextCursor::setPosition: Position '-1' out of range
[02:06:05.262 00-0001188] MTP Info: adding request to toSendMap, msCanWait 0 (session.cpp : 25)
[02:06:05.262 00-0001189] MTP Info: added, requestId 2164 (session.cpp : 25)
[02:06:05.262 00-0001190] MTP Info: dcWithShift 2 stopped send timer, can wait for 0ms from current 903850 (session.cpp : 25)
[02:06:05.262 04-0001191] MTP Info: tryToSend for dc 2. (session_private.cpp : 25)
[02:06:05.262 04-0001192] MTP Info: dc 2 trying to send after ping, state: 2 (session_private.cpp : 25)
[02:06:05.262 04-0001195] MTP Info: sending request, size: 106, num: 529554920, time: 1678755967 (session_private.cpp : 25)
[02:06:05.262 04-0001196] Connection 238(dc:2,149.154.167.41:443) Info: TCP Info: write packet 425 bytes. (connection_abstract.cpp : 25)
[02:06:05.283 04-0001197] Connection 238(dc:2,149.154.167.41:443) Info: Read 457 bytes (connection_abstract.cpp : 25)
[02:06:05.283 04-0001198] Connection 238(dc:2,149.154.167.41:443) Info: Packet received, size = 456. (connection_abstract.cpp : 25)
[02:06:05.283 04-0001199] Connection 238(dc:2,149.154.167.41:443) Info: Decrypted message 7210201973999149057,699,[TRUE] is 420 len (connection_abstract.cpp : 25)
[02:06:05.284 04-0001201] RPC Info: response received for 7210201976759334552, queueing... (session_private.cpp : 25)
[02:06:05.284 04-0001202] RPC Info: gzip container (session_private.cpp : 25)
[02:06:05.284 04-0001203] MTP Info: MTProtoDC::connectionWasInited(), dcWithShift 2 (session.cpp : 25)
[02:06:05.284 04-0001204] Message Info: requests acked, ids [7210201976759334552, 7210201976759334552] (session_private.cpp : 25)
[02:06:05.284 04-0001205] Message Info: removing some old acked sent msgIds 1 (session_private.cpp : 25)
[02:06:05.284 04-0001206] MTP Info: will send 1 acks, ids: [7210201973999149057, 7210201973999149057] (session_private.cpp : 25)
[02:06:05.284 04-0001207] MTP Info: queueTryToReceive() - need to parse in another thread, 1 messages. (session_private.cpp : 25)
[02:06:05.284 00-0001208] MTP Info: dcWithShift 2 can wait for 10000ms from current 0 (session.cpp : 25)
[02:06:05.284 00-0001209] RPC Info: found parser for request 2164, trying to parse response... (mtp_instance.cpp : 25)
[02:06:05.284 00-0001210] MTP Info: unregistering request 2164. (mtp_instance.cpp : 25)
[02:06:06.207 00-0001211] MTP Info: adding request to toSendMap, msCanWait 0 (session.cpp : 25)
[02:06:06.207 00-0001212] MTP Info: added, requestId 2165 (session.cpp : 25)
[02:06:06.207 00-0001213] MTP Info: dcWithShift 2 stopped send timer, can wait for 0ms from current 904472 (session.cpp : 25)
[02:06:06.207 04-0001214] MTP Info: tryToSend for dc 2. (session_private.cpp : 25)
[02:06:06.207 04-0001215] MTP Info: dc 2 trying to send after ping, state: 2 (session_private.cpp : 25)
[02:06:06.207 04-0001218] MTP Info: sending request, size: 98, num: 297114592, time: 1678755968 (session_private.cpp : 25)
[02:06:06.207 04-0001219] Connection 238(dc:2,149.154.167.41:443) Info: TCP Info: write packet 393 bytes. (connection_abstract.cpp : 25)
[02:06:06.228 04-0001220] Connection 238(dc:2,149.154.167.41:443) Info: Read 89 bytes (connection_abstract.cpp : 25)
[02:06:06.228 04-0001221] Connection 238(dc:2,149.154.167.41:443) Info: Packet received, size = 88. (connection_abstract.cpp : 25)
[02:06:06.229 04-0001222] Connection 238(dc:2,149.154.167.41:443) Info: Decrypted message 7210201978059452417,701,[TRUE] is 48 len (connection_abstract.cpp : 25)
[02:06:06.229 04-0001224] RPC Info: response received for 7210201980821855512, queueing... (session_private.cpp : 25)
[02:06:06.229 04-0001225] MTP Info: MTProtoDC::connectionWasInited(), dcWithShift 2 (session.cpp : 25)
[02:06:06.229 04-0001226] Message Info: requests acked, ids [7210201980821855512, 7210201980821855512] (session_private.cpp : 25)
[02:06:06.229 04-0001227] Message Info: removing some old acked sent msgIds 1 (session_private.cpp : 25)
[02:06:06.229 04-0001228] MTP Info: will send 1 acks, ids: [7210201978059452417, 7210201978059452417] (session_private.cpp : 25)
[02:06:06.229 04-0001229] MTP Info: queueTryToReceive() - need to parse in another thread, 1 messages. (session_private.cpp : 25)
[02:06:06.229 00-0001230] MTP Info: dcWithShift 2 can wait for 10000ms from current 0 (session.cpp : 25)
[02:06:06.230 00-0001231] RPC Info: found parser for request 2165, trying to parse response... (mtp_instance.cpp : 25)
[02:06:06.230 00-0001232] MTP Info: unregistering request 2165. (mtp_instance.cpp : 25)
[02:06:06.237 00-0001233] MTP Info: adding request to toSendMap, msCanWait 0 (session.cpp : 25)
[02:06:06.237 00-0001234] MTP Info: added, requestId 2166 (session.cpp : 25)
[02:06:06.237 00-0001235] MTP Info: dcWithShift 2 stopped send timer, can wait for 0ms from current 905417 (session.cpp : 25)
[02:06:06.237 04-0001236] MTP Info: tryToSend for dc 2. (session_private.cpp : 25)
[02:06:06.237 04-0001237] MTP Info: dc 2 trying to send after ping, state: 2 (session_private.cpp : 25)
[02:06:06.237 04-0001240] MTP Info: sending request, size: 94, num: 426757504, time: 1678755968 (session_private.cpp : 25)
[02:06:06.237 04-0001241] Connection 238(dc:2,149.154.167.41:443) Info: TCP Info: write packet 377 bytes. (connection_abstract.cpp : 25)
[02:06:06.251 00-0001242] MTP Info: adding request to toSendMap, msCanWait 0 (session.cpp : 25)
[02:06:06.251 00-0001243] MTP Info: added, requestId 2167 (session.cpp : 25)
[02:06:06.251 00-0001244] MTP Info: dcWithShift 2 stopped send timer, can wait for 0ms from current 0 (session.cpp : 25)
[02:06:06.251 04-0001245] MTP Info: tryToSend for dc 2. (session_private.cpp : 25)
[02:06:06.251 04-0001246] MTP Info: dc 2 trying to send after ping, state: 2 (session_private.cpp : 25)
[02:06:06.251 04-0001249] MTP Info: sending request, size: 82, num: 484129236, time: 1678755968 (session_private.cpp : 25)
[02:06:06.251 04-0001250] Connection 238(dc:2,149.154.167.41:443) Info: TCP Info: write packet 329 bytes. (connection_abstract.cpp : 25)
[02:06:06.267 04-0001251] Connection 238(dc:2,149.154.167.41:443) Info: Read 89 bytes (connection_abstract.cpp : 25)
[02:06:06.267 04-0001252] Connection 238(dc:2,149.154.167.41:443) Info: Packet received, size = 88. (connection_abstract.cpp : 25)
[02:06:06.267 04-0001253] Connection 238(dc:2,149.154.167.41:443) Info: Decrypted message 7210201978221624321,703,[TRUE] is 48 len (connection_abstract.cpp : 25)
[02:06:06.267 04-0001255] RPC Info: response received for 7210201980951491120, queueing... (session_private.cpp : 25)
[02:06:06.267 04-0001256] MTP Info: MTProtoDC::connectionWasInited(), dcWithShift 2 (session.cpp : 25)
[02:06:06.267 04-0001257] Message Info: requests acked, ids [7210201980951491120, 7210201980951491120] (session_private.cpp : 25)
[02:06:06.267 04-0001258] Message Info: removing some old acked sent msgIds 1 (session_private.cpp : 25)
[02:06:06.267 04-0001259] MTP Info: will send 1 acks, ids: [7210201978221624321, 7210201978221624321] (session_private.cpp : 25)
[02:06:06.267 04-0001260] MTP Info: queueTryToReceive() - need to parse in another thread, 1 messages. (session_private.cpp : 25)
[02:06:06.267 00-0001261] MTP Info: dcWithShift 2 can wait for 10000ms from current 0 (session.cpp : 25)
[02:06:06.267 00-0001262] RPC Info: found parser for request 2166, trying to parse response... (mtp_instance.cpp : 25)
[02:06:06.267 00-0001263] MTP Info: unregistering request 2166. (mtp_instance.cpp : 25)
[02:06:06.286 04-0001264] Connection 238(dc:2,149.154.167.41:443) Info: Read 105 bytes (connection_abstract.cpp : 25)
[02:06:06.286 04-0001265] Connection 238(dc:2,149.154.167.41:443) Info: Packet received, size = 104. (connection_abstract.cpp : 25)
[02:06:06.286 04-0001266] Connection 238(dc:2,149.154.167.41:443) Info: Decrypted message 7210201978306327553,705,[TRUE] is 60 len (connection_abstract.cpp : 25)
[02:06:06.286 04-0001268] RPC Info: response received for 7210201981008951764, queueing... (session_private.cpp : 25)
[02:06:06.286 04-0001269] MTP Info: MTProtoDC::connectionWasInited(), dcWithShift 2 (session.cpp : 25)
[02:06:06.286 04-0001270] Message Info: requests acked, ids [7210201981008951764, 7210201981008951764] (session_private.cpp : 25)
[02:06:06.286 04-0001271] Message Info: removing some old acked sent msgIds 1 (session_private.cpp : 25)
[02:06:06.286 04-0001272] MTP Info: will send 2 acks, ids: [7210201978221624321, 7210201978221624321, 7210201978306327553] (session_private.cpp : 25)
[02:06:06.286 04-0001273] MTP Info: queueTryToReceive() - need to parse in another thread, 1 messages. (session_private.cpp : 25)
[02:06:06.286 00-0001274] MTP Info: dcWithShift 2 can wait for 9980ms from current 905455 (session.cpp : 25)
[02:06:06.286 00-0001275] RPC Info: found parser for request 2167, trying to parse response... (mtp_instance.cpp : 25)
[02:06:06.286 00-0001276] MTP Info: unregistering request 2167. (mtp_instance.cpp : 25)
[02:06:06.786 00-0001277] QTextCursor::setPosition: Position '-1' out of range
[02:06:06.786 00-0001278] QPainter::begin: Paint device returned engine == 0, type: 2
[02:06:06.786 00-0001279] QWidget::render: Cannot render with an inactive painter
```

**Comment by uurha** [comment]
Any info about my issue?

**Comment by 23rd** [comment]
What's about behavior of Telegram on mobile OS?

**Comment by uurha** [comment]
On mobile version is correct. And you can see a screenshot of this in `Expected behaviour`

**Comment by uurha** [comment]
Seems fixed in 4.15.1 x64 Win OS
